public/javascripts/data.js

```diff
@@ -394,9 +403,20 @@ var dataNS = odkmaker.namespace.load('odkmaker.data');
                 binding.attrs['jr:preload']='property';
                 binding.attrs['jr:preloadParams']='phonenumber';
             }
+            else if (kind == 'start geopoint')
+            {
+                binding.attrs.type='geopoint';
```

Comment:
and since you have to add that `;` anyway i'll nitpick that i'd like ` = ` w spaces. and same just a few lines down. if not that `;` i'd have just let it go.

Reply:
Added whitespaces around `=` from [lines 358 down to 408](https://github.com/getodk/build/blob/master/public/javascripts/data.js#L351-L398). My editor must have discarded them, upstream still has them all.

Reply:
Missing whitespaces fixed by https://github.com/getodk/build/commit/c05881d8ded6a46d04cc21dedf115e8a74290d0c